app/javascript/sagas/index.js

```diff
@@ -0,0 +1,9 @@
+import { takeEvery } from 'redux-saga/effects'
+
+function* sampleSaga() {
+  yield put(`WIP`)
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
'put' is not defined. (no-undef)